ansible_collections/arista/avd/roles/eos_designs/python_modules/underlay/loopback_interfaces.py

```diff
@@ -47,7 +47,11 @@ def loopback_interfaces(self) -> list | None:
             loopback0["mpls"] = {"ldp": {"interface": True}}
 
         if self.shared_utils.underlay_isis:
-            isis_config = {"isis_enable": self.shared_utils.isis_instance_name, "isis_passive": True}
+            isis_config = {
+                "isis_enable": self.shared_utils.isis_instance_name,
+                "isis_passive": True,
+                "isis_bfd": get(self._hostvars, "underlay_isis_bfd_enable"),
```

Comment:
add default value (in other files as well)

Reply:
I don't think we want a default value here, since the default of `None` will make us remove the field below in line 62.